server/storage/interface.go

```diff
@@ -39,4 +39,15 @@ type MetaStore interface {
 	// Delete removes all metadata for a given GUN.  It does not return an
 	// error if no metadata exists for the given GUN.
 	Delete(gun string) error
+
+	// GetChanges returns an ordered slice of changes. It starts from
+	// the change matching changeID, but excludes this change from the results
+	// on the assumption that if a user provides an ID, they've seen that change.
+	// If changeID is 0, it starts from the
```

Comment:
Hmmm, I thought we were going to keep the latest indicator when we decided to not use the checksums as indices because "delete" wouldn't have a checksum.

And yes, an empty change should also mean "beginning", will fix that.